packages/create-tina-app/src/index.ts

```diff
@@ -76,38 +104,25 @@ export const run = async () => {
   }
   const dirName = projectName
 
-  // If there is no --example passed thought the CLI
-  if (!example) {
-    const res = await prompts({
-      name: 'example',
+  if (!template) {
+    const templateRes = await prompts({
+      name: 'template',
       type: 'select',
       message: 'What starter code would you like to use?',
-      choices: EXAMPLES,
+      choices: TEMPLATES,
     })
-
-    if (typeof res.example !== 'string') {
-      console.error(chalk.red('Input must be a string'))
-      exit(1)
-    }
-    example = res.example
+    template = templateRes.template
   }
-  const chosenExample = EXAMPLES.find((x) => x.value === example)
 
-  if (!chosenExample) {
-    console.error(
-      `The example provided is not a valid example. Please provide one of the following; ${EXAMPLES.map(
-        (x) => x.value
-      )}`
-    )
-  }
   //TODO: Update this?
   await telemetry.submitRecord({
     event: {
       name: 'create-tina-app:invoke',
-      example,
-      useYarn: Boolean(res.packageManager === 'yarn'),
+      example: template,
+      useYarn: Boolean(pkgManager === 'yarn'),
     },
   })
```

Comment:
not sure what we should do wit this... From my understanding this endpoint isn't live. I think we can remove this. Thoughts? @wicksipedia

Reply:
we do get telemetry when the cli is invoked

I'd want to know which package manager is used (as opposed to just useYarn)

Reply:
Done - updated the metrics call to pass in the package manager instead of a boolean